webknossos/webknossos/dataset/_utils/pims_images.py

```diff
@@ -290,48 +290,85 @@ def _open_images(
-                    if isinstance(original_images, Path):
-                        original_images = str(original_images)
+                    try:
+                        if isinstance(original_images[0], Path):
+                            original_images = [str(i) for i in original_images]
+                    except Exception as e2:
+                        pims_open_exceptions.append(e2)
+                if not self._use_bioformats:
                     try:
                         open_kwargs = {}
                         if self._czi_channel is not None:
                             open_kwargs["czi_channel"] = self._czi_channel
                         images_context_manager = pims.open(original_images)
-                    except Exception:
-                        images_context_manager = pims.ImageSequence(original_images)
+                    except Exception as e2:
+                        pims_open_exceptions.append(e2)
+                        try:
+                            images_context_manager = pims.ImageSequence(original_images)
+                        except Exception as e3:
+                            pims_open_exceptions.append(e3)
+                            self._use_bioformats = True
+
+                if self._use_bioformats:
+                    try:
+                        # There is a wrong warning about jpype, supressing it here.
+                        # See issue https://github.com/soft-matter/pims/issues/384
+                        warnings.filterwarnings(
+                            "ignore",
+                            "Due to an issue with JPype 0.6.0, reading is slower.*",
+                            category=UserWarning,
+                            module="pims.bioformats",
+                        )
+                        try:
+                            pims.bioformats._find_jar()
+                        except HTTPError:
+                            # We cannot use the newest bioformats version,
+                            # since it does not include the necessary loci_tools.jar.
+                            # Updates to support newer bioformats jars with pims are in PR
+                            # https://github.com/soft-matter/pims/pull/403
+                            pims.bioformats.download_jar(version="6.7.0")
+                        if "*" in str(original_images) or isinstance(
+                            original_images, list
+                        ):
+                            images_context_manager = pims.ReaderSequence(
+                                original_images, pims.bioformats.BioformatsReader
+                            )
+                        else:
+                            images_context_manager = pims.bioformats.BioformatsReader(
+                                original_images
+                            )
+                    except Exception as e2:
+                        if len(pims_open_exceptions) == 0:
+                            raise
```

Comment:
```suggestion
                            raise e2
```

I find that clearer.